documentation/modules/ROOT/pages/connectors/db2.adoc

```diff
@@ -122,21 +129,225 @@ endif::product[]
+  After the snapshot is complete, and the connector transitions to streaming, it emits change events for any tables that were created after the snapshot began.
+
+.. Produces a `read` event for each row that is captured from a table.
+  All `read` events contain the same LSN position, which is the LSN position that was obtained in step 4.
+
+.. Emits each `read` event to the Kafka topic for the source table.
+
+.. Releases data table locks, if applicable.
+
+8. Record the successful completion of the snapshot in the connector offsets.
+
+The resulting initial snapshot captures the current state of each row in the captured tables.
+From this baseline state, the connector captures subsequent changes as they occur.
+
+After the snapshot process begins, if the process is interrupted due to connector failure, rebalancing, or other reasons, the process restarts after the connector restarts.
+
+After the connector completes the initial snapshot, it continues streaming from the position that it read in Step 4 so that it does not miss any updates.
+
+If the connector stops again for any reason, after it restarts, it resumes streaming changes from where it previously left off.
+
+// ModuleID: {context}-description-of-why-initial-snapshots-capture-the-schema-history-for-all-tables
+// Title: Description of why initial snapshots capture the schema history for all tables
+// Type: concept
+[id="understanding-why-initial-snapshots-capture-the-schema-history-for-all-tables"]
+==== Understanding why initial snapshots capture the schema history for all tables
+
+The initial snapshot that a connector runs captures two types of information:
+
+Table data::
+Information about `INSERT`, `UPDATE`, and `DELETE` operations in tables that are named in the connector's xref:{context}-property-snapshot-include-collection-list[`snapshot.include.collection.list`] property.
+Schema data::
+DDL statements that describe the structural changes that are applied to tables.
+Schema data is persisted to both the internal schema history topic, and to the connector's schema change topic, if one is configured.
+
+After you run an initial snapshot, you might notice that the snapshot captures schema information for tables that are not designated for capture.
+By default, initial snapshots are designed to capture schema information for every table that is present in the database, not only from tables that are designated for capture.
+Connectors require that the table's schema is present in the schema history topic before they can capture a table.
+By enabling the initial snapshot to capture schema data for tables that are not part of the original capture set, {prodname} prepares the connector to readily capture event data from these tables should that later become necessary.
+
+For tables that are designated for capture in the `snapshot.include.collection.list`, the snapshot captures only the table structure.
```

Comment:
I am somewhat concerned about our usage of `snapshot.include.collection.list`.  This information isn't wrong, but it's not complete either.  A user does not have to provide this property and when they do not, we derive the list of tables that are to be snapshot based on the include/exclude list configurations.  This is done by gathering all tables the user can access and then applying the filters to generate a finite list of "capturable" tables.  

A user may opt to specify a `collection.include.collections.list` property if they want to take a snapshot but perhaps they want to capture changes for tables A and B but only want to capture the initial data for B.

If this phrasing is used in other docs, we should be consistent there too.

Reply:
OK, point taken. I had initially mentioned the include/exclude list configurations, but then I realized that _technically_ the list of tables specified in those lists could be overridden by the `include.collections.list`.  So, in your view, in this context, would it be best to keep it simple and not mention the `include.collections.list`?

Reply:
Probably.  Given a configuration as follows:
```
table.include.list=inventory.table1,inventory.table2
snapshot.include.collection.list=inventory.table2
```
In this case, only `table2` would have its data initially snapshot.  If this `snapshot.include.*` property wasn't defined, then the connector snapshots the data of both included tables.  So if we want to include something about this snapshot prefixed property, perhaps a small admonition would suffice?